src/router.js

```diff
@@ -74,6 +74,11 @@ export default new Router({
       path: "/history",
       name: "history",
       component: () => import("./views/History.vue")
+    },
+    {
+      path: "*",
+      name: "error",
```

Comment:
we might have multiple errors so let us name it differently
```suggestion
      name: "error404",
```